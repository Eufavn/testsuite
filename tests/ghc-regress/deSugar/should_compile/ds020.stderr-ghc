
ds020.hs:8:1:
    Warning: Pattern match(es) are overlapped
             In the definition of `a': a ~(~[], ~[], ~[]) = ...

ds020.hs:11:1:
    Warning: Pattern match(es) are overlapped
             In the definition of `b': b ~(~x : ~xs : ~ys) = ...

ds020.hs:16:1:
    Warning: Pattern match(es) are overlapped
             In the definition of `d':
                 d ~(n+43) = ...
                 d ~(n+999) = ...

ds020.hs:22:1:
    Warning: Pattern match(es) are overlapped
             In the definition of `f': f x@(~[]) = ...
