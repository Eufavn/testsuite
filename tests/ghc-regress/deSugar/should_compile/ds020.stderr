
tests/ghc-regress/deSugar/should_compile/ds020.hs:5: Warning: Pattern match(es) are overlapped
    In the definition of `a': a ~(~[], ~[], ~[]) = ...

tests/ghc-regress/deSugar/should_compile/ds020.hs:8: Warning: Pattern match(es) are overlapped
    In the definition of `b': b ~(~x : (~xs : ~ys)) = ...

tests/ghc-regress/deSugar/should_compile/ds020.hs:13: Warning: Pattern match(es) are overlapped
    In the definition of `d':
	d ~(n+43) = ...
	d ~(n+999) = ...

tests/ghc-regress/deSugar/should_compile/ds020.hs:19: Warning: Pattern match(es) are overlapped
    In the definition of `f': f (x@~[]) = ...
